packages/core/src/fhir-to-cda/cda-templates/constants.ts

```diff
@@ -47,3 +47,6 @@ export const codeSystemAttribute = "@_codeSystem";
 export const codeSystemNameAttribute = "@_codeSystemName";
 export const displayNameAttribute = "@_displayName";
 export const namespaceAttribute = "@_xmlns";
+export const xsiTypeAttribute = "@_xsi:type";
+export const xmlnsXsiAttribute = "@_xmlns:xsi";
+export const inlineTextAttribute = "#text";
```

Comment:
So with XML Parser/Builder, the default for text attributes is #text, but you can configure it to use something else like here: https://github.com/metriport/metriport/blob/031d29a3ce1abcd330c9bfb8dca0902dc8c71f23/packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dr-response.ts#L166

The reason its better to use _text is now we can do normal object access with dot notation instead of brackets. 

So lets change that in Builder. The attribute their is also called textNodeName